apiserver/app/createInvoice_test.go

```
@@ -77,3 +76,36 @@ func TestCreateInvoice(t *testing.T) {
 		t.Errorf("Response was incorrect. We expected %s", invoice)
 	}
 }
+
+func TestRateLimiter(t *testing.T) {
+	t.Cleanup(cleanDb(t))
+	a.cache.FlushAll(ctx)
+	_, sessionToken := createTestUser(t, "")
+
+	var requestBody bytes.Buffer
+	multipartWriter := multipart.NewWriter(&requestBody)
+	multipartWriter.WriteField("format", entity.UblFormat)
+	multipartWriter.WriteField("test", "true")
+
+	invoiceWriter, _ := multipartWriter.CreateFormFile("invoice", "ubl21_invoice.xml")
+	invoice, _ := ioutil.ReadFile("../../xml/ubl21/example/ubl21_invoice.xml")
```

let's not ignore errors and fail test if something goes wrong.